pom.xml

```diff
@@ -14,7 +14,8 @@
 	<name>cios-content-service</name>
 	<description>cios-content-service</description>
 	<properties>
-		<java.version>11</java.version>
+		<java.version>8</java.version>
+		<kafka.version>2.8.1</kafka.version>
```

Comment:
@ArpithaSureshappa  Can you please check how we added kafka version on cb-ext and worklow we added this dependency and please check 
cc: @karthik-tarento 
 <dependency>
            <groupId>org.springframework.kafka</groupId>
            <artifactId>spring-kafka</artifactId>
        </dependency>